src/template_compiler_operators.erl

```diff
@@ -144,7 +145,8 @@ modulo(A, B, Runtime, Context) ->
         {undefined, _} -> undefined;
         {_, undefined} -> undefined;
         {_, 0} -> undefined;
-        {_, 0.0} -> undefined;
+        {_, +0.0} -> undefined;
+        {_, -0.0} -> undefined;
```

Comment:
I did that, but I received a warning that it would not match because of the previous clause, so I removed it.
See https://github.com/zotonic/template_compiler/actions/runs/8846752391/job/24293254801?pr=49#step:4:34

Reply:
Yes, though in OTP 27 we need it.
So a bit of a bummer for the warnings in the current OTP versions.

Reply:
I think I just leave it with a warning. This will fix itself over time.